themes/light/assets/js/config/index.js

```diff
@@ -10,6 +10,7 @@ MEDUSA.config.index = function() {
                 name: $(this).val()
             }
         }).then(function(response) {
+            alert('To properly finalize the switch to the new theme, please restart Medusa.');
```

Comment:
Does this mean we can only use a single theme at a time? Can we not save the theme selection in their session and just load it based on that?

Reply:
@p0psicles

Reply:
Yes since we need to load all the mako templates. And map them to routes. There is no other way.

Reply:
I think it could be accomplished if we disable template caching. As it will recompile the mako templates on every requests. This will be a hit on performance. but imo performance trumps functionality.

Reply:
Any idea if we can switch this in runtime? Like can we switch the template path, clear cache and then let it fill the cache again?